### PR TITLE
Fixing label issue in pycbc_page_snrifar

### PR DESCRIPTION
**Pull request by KoustavChandra**
Fixed the overlapping secondary y-ticks issue and forced to display the secondary y-label

### CODE REVIEW

**Comment by GarethDaviesGW** [comment]
I'll do some inline comments for a more detailed review. Could you add some examples (from public data) of the old and new output to this page?

File: bin/hdfcoinc/pycbc_page_snrifar [hunk 1]
```diff
@@ -328,8 +328,17 @@ if not args.cumulative:
...
     pylab.sca(ax1)
+
+pylab.xlabel(r'Coincident Ranking Statistic')
+pylab.yscale('log')
+pylab.ylim(plot_ymin, plot_ymax * 10.0)
+pylab.xlim(plot_xmin, plot_xmax)
+pylab.legend(loc="upper right", fontsize=9)
+pylab.grid()
     
+# No overlapping still now
```
[the remark this answers is not in the document]
Comment (KoustavChandra) [reply bin/hdfcoinc/pycbc_page_snrifar]: It was not needed. I was keeping a track where the issue was rising. Removed it now.

**Comment by KoustavChandra** [comment]
Swapping the tex statement worked. Idea stolen from https://matplotlib.org/3.1.1/tutorials/text/usetex.html

**Comment by KoustavChandra** [comment]
Please find the images [here](https://ligo.gravity.cf.ac.uk/~koustav.chandra/LSC/images/far_vs_rank/)

**Pull request merged**